From: greg@kroah.com
Subject: [PATCH] USB: fix sparse bitwise warnings in the sisusb.c driver
ChangeSet 1.2020.1.2, 2005/03/07 22:16:31-08:00, greg@kroah.com

[PATCH] USB: fix sparse bitwise warnings in the sisusb.c driver

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


 drivers/usb/misc/sisusbvga/sisusb.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)


diff -Nru a/drivers/usb/misc/sisusbvga/sisusb.c b/drivers/usb/misc/sisusbvga/sisusb.c
--- a/drivers/usb/misc/sisusbvga/sisusb.c	2005-03-08 16:56:46 -08:00
+++ b/drivers/usb/misc/sisusbvga/sisusb.c	2005-03-08 16:56:46 -08:00
@@ -565,7 +565,7 @@
 {
 	int ret;
 	int bytes_transferred = 0;
-	u32 tmp;
+	__le32 tmp;
 
 	if (len == 6)
 		packet->data = 0;
@@ -602,7 +602,7 @@
 {
 	int ret;
 	int bytes_transferred = 0;
-	u32 tmp;
+	__le32 tmp;
 
 	if (len == 6)
 		packet->data = 0;
